src/components/new-request/search.vue

```diff
@@ -564,17 +542,29 @@ export default class NewSearch extends Mixins(CommonMixin) {
     this.setNoCorpNum(false)
   }
 
-  @Watch('request_action_cd')
-  watchRequestActionCd (newVal: NrRequestActionCodes) {
-    // Set default location to BC for the requests where BC is the only location option
-    if ([NrRequestActionCodes.CONVERSION, NrRequestActionCodes.MOVE].includes(newVal)) {
+  /** Called when Request Action menu item is changed. */
+  onRequestActionChange (text: string): void {
+    this.request = RequestActions.find(request => request.text === text)
+
+    // clear Jurisdiction and Entity Type
+    this.setLocation(null)
+    if (this.entity_type_cd) {
+      this.entity_type_cd = null
+    }
+    if (this.request?.value !== NrRequestActionCodes.NEW_BUSINESS) {
+      this.setExtendedRequestType(this.request)
+    }
+
+    this.setRequestAction(this.request?.value || null)
+
+    // set default location to BC for requests where BC is the only location option
+    if (this.isNewBcBusiness || this.isContinuationIn || this.isAlterType) {
       this.setLocation(Location.BC)
-      this.locationDisabled = true
```

Comment:
This is now showJurisdiction and it's now a local getter (computed) instead of a variable.